pkg/providerConfigs/detokenize.go

```diff
@@ -32,6 +34,10 @@ func ToTemplateVars(input string, instance Tokens) string {
 		val := value.Field(i) // Use value.Field(i) instead of value.FieldByName(field.Name)
 
 		if normalizedName == strings.ToLower(field.Name) {
+			// TODO: Remove this check once we have a better way to handle empty values.
+			// This is a workaround for the fact that the value of the field cert manager annotations
+			// fields could be empty when cloudflare is used as the origin issuer.
+			// Additionally, it still leaves empty lines in the generated files.
```

Comment:
if it let empty, can we not used the same thing as `helm` with the `dash`  ? 

eg: {{- .Values.something }}`

Reply:
Love it! Pushing it up now. This is why went ahead and opened in Draft.

Reply:
tbh, dunno if it's from `helm` engine or from `gotemplate`  did it work on your test ?

Reply:
strange, it should work because it from the gotemplate engine
https://pkg.go.dev/text/template#hdr-Actions <= check here


can you please provided more explicit example ? =) ( don't totally understand what is done)